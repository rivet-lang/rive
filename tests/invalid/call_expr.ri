fn main() void {
    zet(white, simple, clear);
    xx("", 5, name: 5, fallout: true);

    qy(first: 5, last: 10, 9); // FAIL
}
