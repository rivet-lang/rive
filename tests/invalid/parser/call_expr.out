tests/failing/parser/call_expr.ri:5:28: error: expected named argument, found single expression
    |
  5 |     qy(first: 5, last: 10, 9); // FAIL
    |                            ^
rivetc: error: could not compile package `core`, aborting due to previous error